cpp/log/tree_signer-inl.h

```diff
@@ -193,13 +193,16 @@ util::Status TreeSigner<Logged>::SequenceNewEntries() {
   for (const auto& s : sequenced_hashes) {
     if (!s.second.second /*present*/) {
       // if it disappeared, check it's underwater:
-      CHECK_LT(s.second.first, serving_tree_size);
+      // This is safe because the tree cannot get large enough to overflow a
```

Comment:
How about making `serving_tree_size` an `int64_t` and doing a `CHECK_LE(serving_sth....tree_size(), INT64_MAX);` on the line above it?  Can lose these two casts below then.